.github/ISSUE_TEMPLATE/dependency-update.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Dependency Update
+about: Request a dependency be updated
+title: Dependency update request
+labels: ''
+assignees: ''
+
+---
+
+**Checklist**
+* [ ] I've searched the issue tracker for similar requests
+* [ ] I've confirmed I've run `cargo update`
+* [ ] I've read the Cargo docs on [specifying dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio)
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
I feel like we should skip a bunch of these questions and maybe state explicitly that we're only interested in semver-incompatible bump requests (and maybe quickly explain what that is given that this is like a honeypot for unsophisticated users).

Reply:
@djc WDYT about the revision I just pushed? It adds more explicit guidance/explanation in a comment section that won't be rendered if left-in place by the user before submitting.

I left some question/checkbox content because I think it's still helpful to have. If you feel strongly I can revisit.

Reply:
Seems okay for now.